doc/main.html

```diff
@@ -1622,6 +1655,259 @@ <h3>Complex Example</h3>
+        </p>
+
+        <div class="example">
+          <p>
+            Lower Case Letter:
+          </p> 
+          <pre>
+&lt;ol type="a"&gt;
+  &lt;li&gt;First list item&lt;/li&gt;
+  &lt;li&gt;Second list item&lt;/li&gt;
+&lt;/ol&gt;</pre>
+          
+          <p>is rendered as:</p>
+          <ol type="a">
+            <li>First list item</li>
+            <li>Second list item</li>
+          </ol>
+        </div>
+
+        <div class="example">
+          <p>
+            Upper Case Letter:
+          </p> 
+          <pre>
+&lt;ol type="A"&gt;
+  &lt;li&gt;First list item&lt;/li&gt;
+  &lt;li&gt;Second list item&lt;/li&gt;
+&lt;/ol&gt;</pre>
+          
+          <p>is rendered as:</p>
+          <ol type="A">
+            <li>First list item</li>
+            <li>Second list item</li>
+          </ol>
+        </div>
+
+        <div class="example">
+          <p>
+            Lower Case Roman Numeral:
+          </p> 
+          <pre>
+&lt;ol type="i"&gt;
+  &lt;li&gt;First list item&lt;/li&gt;
+  &lt;li&gt;Second list item&lt;/li&gt;
+&lt;/ol&gt;</pre>
+          
+          <p>is rendered as:</p>
+          <ol type="i">
+            <li>First list item</li>
+            <li>Second list item</li>
+          </ol>
+        </div>
+
+        <div class="example">
+          <p>
+            Upper Case Roman Numeral:
+          </p> 
+          <pre>
+&lt;ol type="I"&gt;
+  &lt;li&gt;First list item&lt;/li&gt;
+  &lt;li&gt;Second list item&lt;/li&gt;
+&lt;/ol&gt;</pre>
+          
+          <p>is rendered as:</p>
+          <ol type="I">
+            <li>First list item</li>
+            <li>Second list item</li>
+          </ol>
+        </div>
+
+        <div class="example">
+          <p>
+            List starting from (3):
+          </p> 
+          <pre>
+&lt;ol start="3"&gt;
+  &lt;li&gt;First list item&lt;/li&gt;
+  &lt;li&gt;Second list item&lt;/li&gt;
+&lt;/ol&gt;</pre>
+          
+          <p>is rendered as:</p>
+          <ol start="3">
+            <li>First list item</li>
+            <li>Second list item</li>
+          </ol>
+        </div>
+
+      </section>
+
+    </section>
+    <section id="sec-list-nesting">
+      <h3>List Nesting</h3>
+
+      <p>
+        Both <code>ol</code> and <code>ul</code> list may be nexted as many times within a <code>li</code> element as needed to achieve the required results. See below for various examples. 
+      </p>
+
+      <div class="example">
+        <p>
+          Nesting <code>ol</code>:
+        </p> 
+        <pre>
+&lt;ol&gt;
+  &lt;li&gt;First list item
+    &lt;ol&gt;
+      &lt;li&gt;First sub item&lt;/li&gt;
+      &lt;li&gt;Second sub item&lt;/li&gt;
+    &lt;/ol&gt;
+  &lt;/li&gt;
+  &lt;li&gt;Second list item&lt;/li&gt;
+&lt;/ol&gt;</pre>
+        
+        <p>is rendered as:</p>
+        <ol>
+          <li>First list item
+            <ol>
+              <li>First sub item</li>
```

Comment:
We should recommend using a different labeling scheme for successive levels.

Reply:
Ah, such as top level 1, 2nd A, 3rd I, etc?

Reply:
Done in https://github.com/SMPTE/html-pub/pull/155/commits/176216373b289729496240bf05a0c5f7cb41a821